checker/manual/figures/value-subtyping.svg

```diff
@@ -425,151 +434,273 @@
+     id="path3546"
+     inkscape:connector-type="polyline"
+     inkscape:connector-curvature="3"
+     inkscape:connection-start="#g24-05"
+     inkscape:connection-end="#g24-05" />
+  <g
+     id="g34-2-5-5-9"
+     transform="matrix(1,0,0,0.86985556,712.01656,-115.89132)">
+    <rect
+       style="fill:#ffffff"
+       x="302.44049"
+       y="320"
+       width="176"
+       height="38"
+       id="rect36-3-8-1-1" />
+    <rect
+       style="fill:none;stroke:#000000;stroke-width:2.14440465"
+       x="302.44049"
+       y="320"
+       width="176"
+       height="38"
+       id="rect38-9-1-7-2" />
+    <text
+       font-size="12.8"
+       style="font-style:normal;font-weight:normal;font-size:14.10753822px;font-family:'courier new';text-anchor:middle;fill:#000000"
+       x="390.44049"
+       y="342.89999"
+       id="text40-7-1-0-7">
+      <tspan
+         x="390.44049"
+         y="342.89999"
+         id="tspan42-4-7-0-0"
+         style="font-size:14.10753822px">@IntRange(int, int)</tspan>
+    </text>
+  </g>
+  <g
+     id="g60-8"
+     transform="matrix(-1,0,0,0.86985538,902.13044,26.085562)">
+    <line
+       style="fill:none;stroke:#000000;stroke-width:3.21660709"
+       x1="402.10901"
+       y1="399.461"
+       x2="302.76199"
+       y2="362.052"
+       id="line62-7" />
+    <polygon
+       style="fill:#000000"
+       points="301,366.732 304.524,357.373 293.404,358.529 "
+       id="polygon64-9" />
+    <polygon
+       style="fill:none;stroke:#000000;stroke-width:3.21660709"
+       points="304.524,357.373 293.404,358.529 301,366.732 "
+       id="polygon66-2" />
+  </g>
+  <g
+     id="g68-0"
+     transform="matrix(1,0,0,0.86985556,205.67461,164.80041)">
+    <line
+       style="fill:none;stroke:#000000;stroke-width:3.21660709"
+       x1="400"
+       y1="320"
+       x2="302.71799"
+       y2="282.168"
+       id="line70-2" />
+    <polygon
+       style="fill:#000000"
+       points="293.398,278.544 300.906,286.828 304.53,277.508 "
+       id="polygon72-3" />
+    <polygon
+       style="fill:none;stroke:#000000;stroke-width:3.21660709"
+       points="300.906,286.828 304.53,277.508 293.398,278.544 "
+       id="polygon74-7" />
+  </g>
+  <g
+     id="g34-5"
+     transform="matrix(1,0,0,0.86985556,200.09846,95.59458)">
+    <rect
+       style="fill:#ffffff"
+       x="150"
+       y="320"
+       width="200"
+       height="38"
+       id="rect36-9" />
+    <rect
+       style="fill:none;stroke:#000000;stroke-width:2.14440465"
+       x="150"
+       y="320"
+       width="200"
+       height="38"
+       id="rect38-2" />
+    <text
+       font-size="12.8"
+       style="font-style:normal;font-weight:normal;font-size:14.10753822px;font-family:'courier new';text-anchor:middle;fill:#000000"
+       x="248"
+       y="342.89999"
+       id="text40-2">
+      <tspan
+         x="248"
+         y="342.89999"
+         id="tspan42-8"
+         style="font-size:14.10753822px">@IntRange(from=1,to=2)</tspan>
```

Comment:
Yes, they are related and should be equivalent. However, the bottom half of the diagram, according to the manual, "are examples of additional subtyping relationships". 

If we do need to put on each and every rules in detail, then it won't be a small diagram, e.g. 
@IntVal({1}) => @IntRange(from = 1, to = 2)
@IntVal({1, 2}) => @IntRange(from = 1, to = 2)
@IntRange(from = 1, to = 2) => @IntVal({1, 2})
@IntVal({1}) => @DoubleVal({1.0})
@IntRange(from = 1, to = 2) => @DoubleVal({1.0, 2.0})
@IntRange(from = 1, to = 2) => @IntRange(from = 0, to = 10)
etc.
where "=>" means "is subtype of", and "==" means "is equivalent to"

Any ideas of which would be the priority and should be put on the diagram? 

Thanks and regards,

Jason